tests/clv/models/test_beta_geo.py

```diff
@@ -324,3 +441,34 @@ def test_model_repr(self):
             "\nr~HalfFlat()"
             "\nlikelihood~Potential(f(r,alpha,b,a))"
         )
+
+    def test_save_load_beta_geo(self, data):
```

Comment:
Can you make a generic test util to test save_load for different CLV models? I imagine most of the boilerplate can be reused?

Reply:
This can be left as an open issue if you don't want to do it in this PR

Reply:
I'll open an issue for that, for now let's get them finally merged